vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -349,19 +358,15 @@ type::ContributionPublisherList Unblinded::PrepareAutoContribution(
   }
 
   const double total_votes = static_cast<double>(list.size());
```

Comment:
Since we're doing some name cleanups, it might be a good idea to just change this parameter name to `token_list` so that it's more obvious how many "votes" we are attempting to distribute.

Reply:
Yup, will do.